src/pal/src/locale/unicode.cpp

```diff
@@ -931,10 +931,10 @@ PAL_BindResources(IN LPCSTR lpDomain)
     LPCSTR boundPath = bindtextdomain(lpDomain, coreCLRDirectoryPath);
 
     return boundPath != NULL;
-#else // __APPLE__
-    // UNIXTODO: Implement for OSX if necessary
+#else // HAVE_LIBINTL_H
+    // UNIXTODO: Implement for Unixes without libintl if necessary
```

Comment:
Should this raise a warning? libintl is available in many linux/bsd/solaris distros, it is only that in some minimal installations, it is not installed by default, but we can get it from packages. So if the build machine doesn't have it installed, that doesn't mean intention is to disable this whole feature, which begs a question; should this be optional at all?

Reply:
Currently, we don't have any translations of messages, so this function is just a pass-through in all cases. The reason why it is optional is that we didn't want to use libintl for OSX (unless it turns out it is impossible to use native localization stuff) and for Android. And I was thinking there may be some minor Linux distros that don't have libintl at all.